frontend/awx/views/schedules/wizard/ScheduleReviewStep.tsx

```diff
@@ -0,0 +1,142 @@
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { AwxRoute } from '../../../main/AwxRoutes';
+import { PageDetail, PageDetails, PageWizardStep, useGetPageUrl } from '../../../../../framework';
+import { PageDetailCodeEditor } from '../../../../../framework/PageDetails/PageDetailCodeEditor';
+import { usePageWizard } from '../../../../../framework/PageWizard/PageWizardProvider';
+import { jsonToYaml } from '../../../../../framework/utils/codeEditorUtils';
+import {
+  WizardFormValues,
+  UnifiedJobType,
+} from '../../../resources/templates/WorkflowVisualizer/types';
+import {
+  hasDaysToKeep,
+  getValueBasedOnJobType,
+} from '../../../resources/templates/WorkflowVisualizer/wizard/helpers';
+import { PromptReviewDetails } from '../../../resources/templates/WorkflowVisualizer/wizard/PromptReviewDetails';
+import { RESOURCE_TYPE } from './constants';
+import { ScheduleFormWizard } from '../types';
+import { PageFormSection } from '../../../../../framework/PageForm/Utils/PageFormSection';
+import { RulesPreview } from '../components/RulesPreview';
+import { ExceptionsPreview } from '../components/ExceptionsPreview';
+
+const ResourceLink: Record<UnifiedJobType, AwxRoute> = {
+  inventory_update: AwxRoute.InventorySourceDetail,
+  job: AwxRoute.JobTemplateDetails,
+  project_update: AwxRoute.ProjectDetails,
+  system_job: AwxRoute.ManagementJobDetails,
+  workflow_approval: AwxRoute.WorkflowApprovalDetails,
+  workflow_job: AwxRoute.WorkflowJobTemplateDetails,
+};
+
+export function ScheduleReviewStep() {
+  const { t } = useTranslation();
+  const getPageUrl = useGetPageUrl();
+
+  const { wizardData, visibleSteps } = usePageWizard() as {
+    wizardData: WizardFormValues & ScheduleFormWizard;
+    visibleSteps: PageWizardStep[];
+  };
+  const {
+    approval_name,
+    approval_description,
+    node_type,
+    resource,
+    approval_timeout,
+    node_alias,
+    node_convergence,
+    node_days_to_keep,
+    name,
+    description,
+    startDateTime,
+    timezone,
+    exceptions,
+  } = wizardData;
+
+  const hasPromptDetails = Boolean(visibleSteps.find((step) => step.id === 'nodePromptsStep'));
+  const nodeTypeDetail = useGetNodeTypeDetail(node_type);
+  const nameDetail = getValueBasedOnJobType(node_type, resource?.name || '', approval_name);
+  const descriptionDetail = getValueBasedOnJobType(
+    node_type,
+    resource?.description || '',
+    approval_description
+  );
+  const convergenceDetail = node_convergence === 'all' ? t('All') : t('Any');
+  const timeoutString = useGetTimeoutString(approval_timeout);
+  const timeoutDetail = getValueBasedOnJobType(node_type, '', timeoutString);
```

Comment:
This won't apply to us.  Timeout is a value that lives on workflow approvals and they don't have schedules.